Json/src/main/java/org/tribuo/json/StripProvenance.java

```diff
@@ -280,11 +275,6 @@ public String getOptionsDescription() {
          */
         @Option(charName = 't', longName = "hash-type", usage = "The hash type to use.")
         public ProvenanceUtil.HashType hashType = ObjectProvenance.DEFAULT_HASH_TYPE;
-        /**
-         * Read and write protobuf formatted models.
-         */
-        @Option(longName = "model-protobuf", usage = "Read and write protobuf formatted models.")
```

Comment:
Breaking change

Reply:
I've reverted this to allow this argument to be a no-op as StripProvenance is part of the supported API.